lib/devtools.rb

```diff
@@ -15,6 +20,10 @@
 require 'devtools/project/initializer/rake'
 require 'devtools/project/initializer/rspec'
 
```

Comment:
I think that providing / abstracting these is the domain of devtools. So no grouping should be required.